### PR TITLE
PMM-4145 Add dissable metric collection flags to RDS.

### PR DESCRIPTION
**Pull request by askomorokhov**
https://jira.percona.com/browse/PMM-4145

Ref:
- [ ] https://github.com/percona/pmm/pull/500
- [ ] https://github.com/percona/pmm-managed/pull/342
- [ ] https://github.com/percona/grafana-dashboards/pull/629
- [ ] https://github.com/percona/rds_exporter/pull/41
- [ ] https://github.com/percona/pmm-admin/pull/96

FB:
- [ ]  https://github.com/Percona-Lab/pmm-submodules/pull/750

### CODE REVIEW

File: api/inventorypb/agents.proto [hunk 1]
```diff
@@ -877,6 +885,10 @@ message AddRDSExporterRequest {
   map<string, string> custom_labels = 5;
   // Skip connection check.
   bool skip_connection_check = 6;
+  // Basic metrics are disabled.
+  bool basic_metrics_disabled = 7;
```
Comment (BupycHuk) [inline api/inventorypb/agents.proto]: Why these variables are different from `disable_basic_metrics` and `disable_enhanced_metrics`?
I think that they should be the same or at least swapped.

Comment (AlekSi) [reply api/inventorypb/agents.proto]: They should be swapped. `basic_metrics_disabled` describes the current state. `disable_basic_metrics` describes the needed action.

**Comment by AlekSi** [comment]
Should be merged once the tree is open for 2.5

**Comment by AlekSi** [comment]
CI passed: https://travis-ci.com/github/percona/pmm/builds/154745356

**Pull request merged**